README.md

```diff
@@ -32,6 +32,16 @@ Best practices for using the Office.js NPM package include:
 
 - Use TypeScript definitions for Office.js as described in [IntelliSense definitions](#intellisense-definitions).
 
+### Support
+
+The Office.js CDN is the official, supported source for Office Add-ins. This means that for the NPM package the following are not guaranteed.
+
+- Frequency of updates
+
+- Timely support for issues and bugs
+
+Also, Outlook does not support offline scenarios due to a network access requirement for dependencies like the Microsoft Ajax library.
```

Comment:
Should we say "offline hosting of JavaScript library" instead of "offline scenario"